modules/gatk4/combinegvcfs/main.nf

```diff
@@ -0,0 +1,48 @@
+process GATK4_COMBINEGVCFS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::gatk4=4.2.4.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gatk4:4.2.4.1--hdfd78af_0' :
+        'quay.io/biocontainers/gatk4:4.2.4.1--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta), path(fastaIndex), path (fastaDict)
+    path(vcffiles)
+    path(vcf_idx)
```

Comment:
Shouldn't it be more something like that?
```suggestion
    tuple val(meta), path(fasta), path(vcf), path(vcf_idx)
    path (fastaIndex)
    path (fastaDict)
```

Reply:
the vcf and vcf_idx are lists, not single instances and will be gathered via collect by the user. Would prefer the current input order

Reply:
list of files are still path objects. The fasta does not depend on the meta map though, does it? I mean it is one reference for all samples, but the list of vcf files belongs to the meta map and is used to gather the right vcfs files

Reply:
@FriederikeHanssen -  You are correct in tis instance except the list of vcf files will belong to multiple meta maps. How should we deal with this?

Reply:
Remap the vcf files before this process and give this set a new meta map with an id that will be used as the output name

Reply:
the meta map I guess has a field for the splitting this should be removed. You will need to touch it anyways to group the right vcf files together for the input in the current implementation

Reply:
So the input will be [meta, [list of vcf files], [list of vcf indexes] ], correct?

Reply:
Just meta, vcf, vcf_index is fine, as we're doing in the manta germline module for example that can accommodate multiple inputs: https://github.com/nf-core/modules/blob/master/modules/manta/germline/main.nf#L11

Reply:
Sorry, my question might not have been clear - I was just talking about within the workflow. I understand the rational of this approach - all good here.